### PR TITLE
Add inline comments section

### PR DESCRIPTION
**Pull request by ulyssessouza**
### Related issues (optional)
Resolves https://github.com/docker/compose/issues/9327

### CODE REVIEW

**Comment by ulyssessouza** [comment]
@usha-mandya PTAL

File: compose/compose-file/compose-file-v3.md [hunk 1]
```diff
@@ -1107,6 +1107,16 @@ also ignored.
 RACK_ENV=development
 ```
 
+Compose also recognizes inline comments, like in:
+```
+MY_VAR = value # this is a comment
+```
+
+To avoid interpreting "#" as an inline comment, please use the quoted notation:
```
Comment (usha-mandya) [inline compose/compose-file/compose-file-v3.md:1115]:
```suggestion
To avoid interpreting "#" as an inline comment, use the quotation marks:
```

**Comment by usha-mandya** [comment]
Made a minor suggestion. Looks good.

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**